lib/puppet/settings.rb

```diff
@@ -1062,6 +1062,11 @@ def to_manifest
   # Create the necessary objects to use a section.  This is idempotent;
   # you can 'use' a section as many times as you want.
   def use(*sections)
+    Puppet.warning(":master section deprecated in favor of :server section") if sections.include?(:master)
+
+    # add :server if sections include :master or :master if sections include :server
+    sections |= [:master, :server] if (sections & [:master, :server]).any?
+
```

Comment:
If `sections` includes `:master` then we add `:master` again. Is there an issue if a section is "used" twice? I see later on we `uniq` the used sections, but I'm more concerned about below where we convert the sections to a settings catalog: https://github.com/puppetlabs/puppet/pull/8367/files#diff-feceffda334b8cc3287b33394350b4ec29a27ae6efad4bbd188e9c892f7107acR1078. Do we try to add the same file resource multiple times?